doc/source/doc-style/formatting-tools.rst

```diff
@@ -11,6 +11,19 @@ Most of the tools presented can be configured using :ref:`the
 cleaner root project directory.
 
 
+Blacken-Docs
+------------
+
+When writing documentation, it is frequent to include code-blocks which are used
+as examples. However, these code snippets cannot be tested with the usual code
```

Comment:
@jorgepiloto Using the word "test" was confusing and people could think that the code snippet is executed and tested, while only formatting is verified.
```suggestion
as examples. However, these code snippets style cannot be verified with the usual code
```

Reply:
Thanks for pointing this out, @MaxJPRey.